### ISSUE TITLE
Documentation Bug: Format issue in mxnet.util.use_np document

### ISSUE THREAD

**Issue by DNXie** [apache/mxnet#18988]
## Description
This is a documentation bug. There is a format issue in `mxnet.util.use_np` document Parameter section. `and NumPy-array semantics. ` is not a parameter.

![image](https://user-images.githubusercontent.com/24580222/90995106-eba51900-e588-11ea-8e20-24b523e1d8f3.png)

### Error Message
None

## To Reproduce
Link to the buggy document: https://mxnet.apache.org/versions/1.6/api/python/docs/api/mxnet/util/index.html#mxnet.util.use_np

### Steps to reproduce
(Paste the commands you ran that produced the error.)

1.
2.

## What have you tried to solve it?

1.
2.

## Environment

We recommend using our script for collecting the diagnositc information. Run the following command and paste the outputs below:
```
curl --retry 10 -s https://raw.githubusercontent.com/dmlc/gluon-nlp/master/tools/diagnose.py | python

# paste outputs here
```
OS: ubuntu 18.04
Python: 3.7.6
pip: 20.0.2
numpy: 1.18.5
mxnet: 1.6.0

**Comment by szha** [comment]
The problem is the wrong doc string and indentation here:
https://github.com/apache/incubator-mxnet/blob/3c4ac19daa3e645d918692b864ea19640f7e0314/python/mxnet/util.py#L541-L542

A proper documentation format looks like this:
https://github.com/apache/incubator-mxnet/blob/3c4ac19daa3e645d918692b864ea19640f7e0314/python/mxnet/util.py#L773-L776

**Comment by vijaya-lakshmi-venkatraman** [comment]
Hello,
I see a PR merged for this issue.
Is this still open?

**Comment by DNXie** [comment]
@vijaya-lakshmi-venkatraman  I don't think the issue is fixed.

**Comment by vijaya-lakshmi-venkatraman** [comment]
@DNXie I am looking at the [master ](https://mxnet.apache.org/versions/master/api/python/docs/api/util/index.html#mxnet.util.use_np )version and dont see the second bullet point anymore however the description seems to be incomplete

Am I looking at the right version & the right place?

**Comment by DNXie** [comment]
@vijaya-lakshmi-venkatraman 
Hi, it is fixed in the the master version. But in the [1.6]( https://mxnet.apache.org/versions/1.6/api/python/docs/api/mxnet/util/index.html#mxnet.util.use_np) document, it still exists. Should we consider this as fixed?

**Comment by vijaya-lakshmi-venkatraman** [comment]
@DNXie Isn't master the current live version? Do you prefer to have it fixed in 1.6 branch as well?

**Comment by DNXie** [comment]
@vijaya-lakshmi-venkatraman  It looks good now. I will close the issue.
